style.css

```diff
@@ -43,24 +43,16 @@ body {
   align-items: center;
 }
 
-.end-text{
+.game-over {
   color: #bb0a1e;
   font-size: 42px;
   text-align: center;
   padding-left: 15px;
 }
 
-.is-hidden1{
-  display: none;
-}
-
-.is-hidden2{
-  display: none;
-}
-
-.sub-text{
+.sub-text {
   color: #fff;
-  font-size: 27px;
+  font-size: 28px;
```

Comment:
many people prefer even numbers for size or multiples of 5. I like even nummbers. Since 27 was not even I made it 28px.